content/3_Silver/Two_Pointers.mdx

```diff
@@ -44,15 +44,145 @@ of an interval. It can also be used to track two values in an array as shown in
 
 We want to find the longest contiguous segment of books that can be read within $t$ minutes. 
 
-To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ as the start and end of the segment respectively and use two pointers to track the interval.
+To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ to **represent** the beginning and end of the **segment**. However, both will start at the beginning of the array. These numbers can be thought of as pointers, hence the name "two pointers".
```

Comment:
Any reason why "represent" and "segment" are bolded?

Reply:
I thought it made it more clear